wagtail_wordpress_import/prefilters/apps.py

```diff
@@ -0,0 +1,7 @@
+from django.apps import AppConfig
```

Comment:
Why does this need to be an app?

Reply:
Thanks looks like it doesn't. I will remove that file.